.github/workflows/cd.yml

```diff
@@ -24,7 +24,7 @@ defaults:
 jobs:
```

Comment:
The changes in this file allow us to build the binaries and add them to the most recent release. This workflow is automatically kicked off when a new release is created.